packages/salesforcedx-apex-debugger/src/context/isvContextUtil.ts

```diff
@@ -18,13 +18,11 @@ export class IsvContextUtil {
   public async setIsvDebuggerContext(projectWorkspacePath: string) {
     let isvDebugProject = false;
     if (projectWorkspacePath) {
-      const config = await new ConfigGet().getConfig(
-        projectWorkspacePath,
-        SF_CONFIG_ISV_DEBUGGER_SID,
-        SF_CONFIG_ISV_DEBUGGER_URL
-      );
-      const isvDebuggerSid = config.get(SF_CONFIG_ISV_DEBUGGER_SID);
-      const isvDebuggerUrl = config.get(SF_CONFIG_ISV_DEBUGGER_URL);
+      const configAggregator: ConfigAggregator = await ConfigAggregator.create({
+        projectPath: projectWorkspacePath
+      });
+      const isvDebuggerSid = JSON.stringify(configAggregator.getPropertyValue(SF_CONFIG_ISV_DEBUGGER_SID));
+      const isvDebuggerUrl = JSON.stringify(configAggregator.getPropertyValue(SF_CONFIG_ISV_DEBUGGER_URL));
 
       if (typeof isvDebuggerSid !== 'undefined' && typeof isvDebuggerUrl !== 'undefined') {
         // set auth context
```

Comment:
what do you think the point of `// set auth context` is?

hint: peek at how `setIsvDebuggerContext` is called from `setupGlobalDefaultUserIsvAuth`.  What would happen if those weren't set?

would the reset still be necessary?

Reply:
and the perennial question...why is this a class?

Reply:
how simple could this code be if it only needs to know if the two props are defined to return a boolean?

Reply:
and does it really belong in the debugger, or in the ext that needs it?